Sources/PubNubChat/Storage/CoreData/CoreDataMigrationManager.swift

```diff
@@ -38,23 +38,24 @@ public protocol CoreDataMigrationManager {
 }
 
 extension NSManagedObjectModel {
-  var versionID: Int {
+  var integerValue: Int {
```

Comment:
why the previous name was incorrect?

Reply:
I thought the new variable name is more meaningful but apparently it's not. The idea that stays behind that is setting an integer value in ascending order for each new model version using the built-in [versionIdentifiers](https://developer.apple.com/documentation/coredata/nsmanagedobjectmodel/1506268-versionidentifiers) property. This should facilitate current and future migration processes because we will exactly know what's the current model and what's the model we want to migrate. Let me know if you have any idea how to rename this variable.

Reply:
I do not know this code well so I was just curious, if integerValue is clear for you than for me it is good too :)